lib/transactional.pm

```diff
@@ -158,7 +158,7 @@ sub check_target_version {
     my $release = script_output "cat /etc/os-release";
     my $expected_version = get_var("TARGET_VERSION", get_required_var("VERSION"));
 
-    die "Target version not found! Expected: $expected_version" if ($release !~ "VERSION=\"?$expected_version\"?");
+    die "Target version not found! Expected: $expected_version" if ($release !~ "VARIANT=\"Micro ?$expected_version\"?");
```

Comment:
For 6.2 the change is expected, while for 6.0 & 6.1 we should reserve the old condition.